src/test/java/bio/terra/integration/IngestTest.java

```diff
@@ -412,34 +389,19 @@ public void ingestMergeHappyPathTest() throws Exception {
         "correct merge again sample row count",
         mergeAgainIngestResponse.getRowCount(),
         equalTo(1L));
+    assertSampleTableIdColumnRemainsUnchanged(dataset);
+  }
 
-    bqQueryResult = bigQueryProject.query(bqTableInfo.getSampleQuery());
-    assertThat("Expected number of rows are present", bqQueryResult.getTotalRows(), equalTo(7L));
-
-    results =
-        BQTestUtils.mapToList(
-            bqQueryResult, "id", "participant_ids", "date_collected", "derived_from");
-
-    List<Map<String, Object>> dataPostMergeAgain =
-        jsonLoader.loadObjectAsStream(
-            "ingest-test/merge/ingest-test-sample-merge-again-expected.json",
-            new TypeReference<>() {});
-
+  private void assertSampleTableIdColumnRemainsUnchanged(DatasetModel dataset) throws Exception {
+    int expectedNumRows = 7;
+    dataRepoFixtures.assertDatasetTableCount(steward(), dataset, "sample", expectedNumRows);
+    List<String> actualValues =
+        dataRepoFixtures.retrieveColumnTextValues(steward(), datasetId, "sample", "id");
     assertThat(
-        "Only specified records and fields updated by second merge ingest",
-        results,
+        "Expected values returned from column stats endpoint for sample id table",
+        actualValues,
         containsInAnyOrder(
-            // Newly updated row
-            dataPostMergeAgain.get(0), // ID = sample1
-            // Prior updated row
-            dataPostMerge.get(1), // ID = sample2
-            // Untouched rows
-            dataOriginal.get(2), // ID = sample3
-            dataOriginal.get(3), // ID = sample4
-            dataOriginal.get(4), // ID = sample5
-            dataOriginal.get(5), // ID = sample6
-            dataOriginal.get(6) // ID = sample7
-            ));
+            "sample1", "sample2", "sample3", "sample4", "sample5", "sample6", "sample7"));
```

Comment:
Thank you so much for updating this test!

Reply:
No problem! Hopefully, it added a small bit of test coverage to the original test and allowed me to add test coverage to the new feature without actually adding an integration test! win-win!